Specs/DateTime/English/DateParser.json

```diff
@@ -2044,4 +2044,100 @@
     }
   ]
 }
+,
+{
+  "Input": "We had a meeting 1 month 21 days ago",
```

Comment:
It doesn't support "1 months and 21 days ago"?
Or "1 month, 21 days ago"?

Reply:
What about short forms like "1y 3d"?
And "one week three days"?

Reply:
"1 months and 21 days ago" and "1 month, 21 days ago" are supported.
"one week three days" is supported.
"1y 3d" is not supported. We do not extract "1y" and year. The same for "3d".

Reply:
Can you add test cases for the 3 supported ones?